src/o365/teams/commands/report/teams-report-useractivityuserdetail.ts

```diff
@@ -49,12 +53,48 @@ class TeamsReportUserActivityUserDetailCommand extends GraphCommand {
     request
       .get(requestOptions)
       .then((res: any): void => {
-        cmd.log(res);
 
+        let content: string = '';
+
+        if(args.options.output && args.options.output.toLowerCase() === 'json')
+        {
+          let reportdata = this.getJsonReport(res);
+          content = JSON.stringify(reportdata);
+          cmd.log(reportdata);
+        }
+        else
+        {
+          content = res;
+          cmd.log(content);
```

Comment:
I added extra condition so that log should show in case user has not specified `outputFile` so the code looks like:
```
if (!args.options.outputFile) {
            cmd.log(content);
 }
```

This follows the same pattern we have in other commands like [file-get](https://github.com/pnp/office365-cli/blob/5d2a258229af36de72962ac529d7af11a07a3406/src/o365/spo/commands/file/file-get.ts#L113) where no output on the screen is shown, but just file is generated.